rules/use-the-right-data-storage/rule.md

```diff
@@ -0,0 +1,28 @@
+---
+type: rule
+archivedreason: 
+title: Do you use the right data storage?
+guid: 4f7c2e78-43f9-4638-a85c-2ca22a040f43
+uri: use-the-right-data-storage
+created: 2017-06-28T22:16:46.0000000Z
+authors:
+- title: Luke Cook
+  url: https://ssw.com.au/people/luke-cook
+related:
+- data-lake
+redirects:
+
+---
+
+Databases, data warehouses, and data lakes are all essential components of Business Intelligence (BI) systems, but they serve different purposes and have distinct characteristics. Understanding their differences can help organizations make informed decisions about data storage and analysis.
+
+<!--endintro-->
+
+## Databases
```

Comment:
Is it worth mentioning structured and unstructured and what the difference would be here? There is a bit that says usually tables defining relationships, but these days it is becoming more popular with the types of workloads systems need to handle real-time to use unstructured databases that don't have defined relationships